ksql-api/src/test/java/io/confluent/ksql/api/TestUtils.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
+import org.hamcrest.Matcher;
+
+public class TestUtils {
+
+  public static void awaitLatch(CountDownLatch latch) throws Exception {
+    assertThat(latch.await(2000, TimeUnit.MILLISECONDS), is(true));
+  }
+
+  public static class AsyncAssert {
+
+    private AssertionError error;
+
+    public synchronized <T> void assertAsync(Supplier<? extends T> actualSupplier,
+        Matcher<? super T> expected) {
+      try {
+        assertThat(actualSupplier.get(), expected);
+      } catch (AssertionError e) {
+        error = e;
+      }
+    }
```

Comment:
What's the benefit of accepting a supplier, rather than the actual value, given the code immediately calls `get()` on the supplier, and only calls `get()` once?  Surely this can be simplified to just take the actual value?

Reply:
ack, copy and paste